lucene/translate.py

```diff
@@ -154,36 +159,35 @@ def postprocess_translated_objc(path):
             if any(fnmatch.fnmatch(full_path_java, ptn) for ptn in excluded):
                 continue
 
-            full_path_m = full_path_java.replace(
-                src, dst).replace(".java", ".m")
+            full_path_m = full_path_java.replace(src, dst).replace(".java", ".m")
             if os.path.exists(full_path_m):
                 if os.path.getmtime(full_path_m) >= os.path.getmtime(full_path_java):  # nopep8
                     continue
             to_compile.append(full_path_java)
-            to_postprocess.append(full_path_m)
 
-    print('%s => %s files to compile' % (src, len(to_compile)))
+    print('\nProcessing: %s, %d java files to compile.\n' % (src, len(to_compile)))
 
-    if to_compile:
-        print('Compiling %d java files' % len(to_compile))
+    for file_to_compile in to_compile:
+        print('Compiling: ' + file_to_compile)
         args = [
-            'j2objc',
-            # '-use-arc',
-            '-classpath', classpaths,
-            '--segmented-headers',
-            '-sourcepath', src,
+            j2objc,
             '-d', dst,
+            '-sourcepath', ' '.join(classpaths),
+            '-use-arc', '--swift-friendly', '--nullability', '--doc-comments', '--no-extract-unsequenced', '--segmented-headers', file_to_compile
```

Comment:
My experience is that `-use-arc` is not good for large Java projects such Lucene:

1. Lucene use a lot of exceptions, which is pretty typical for a Java project. Objective-C exceptions under ARC leak memory unless you specify `-fobjc-arc-exceptions`, which incurs extra performance cost. By not using ARC you don't have to build with that extra flag.
2. ARC code can interoperate with non-ARC code.
3. The last point may be more subjective, but I found non-ARC code slightly easier to track down leaks, which was how I found the places that needed the J2ObjC `@Weak` annotations.